src/gui/AutomationPatternView.cpp

```diff
@@ -21,12 +21,12 @@
  * Boston, MA 02110-1301 USA.
  *
  */
+#include "AutomationPatternView.h"
```

Comment:
@Umcaruje moving this up like this... is this just a "THIS IS MY HEADER!!" convention?  This is new to me.